app/controllers/schedules_controller.rb

```diff
@@ -3,6 +3,14 @@ class SchedulesController < ApplicationController
   before_action :respond_to_options
   load_and_authorize_resource :conference, find_by: :short_title
   load_resource :program, through: :conference, singleton: true, except: :index
+  around_action :use_timezone_for_this_request
+
+  # FIXME: The timezome should only be applied on output, otherwise
+  # you get lost in timezone conversions...
+
+  def use_timezone_for_this_request(&block)
```

Comment:
Have you tried this? Because I think it will break the whole schedule as it is implemented to work in UTC.